recipes/ogg/all/conanfile.py

```diff
@@ -53,4 +62,8 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
-        self.cpp_info.libs = ['ogg']
+        self.cpp_info.names["cmake_find_package"] = "Ogg"
+        self.cpp_info.names["cmake_find_package_multi"] = "Ogg"
+        self.cpp_info.components["ogglib"].names["cmake_find_package"] = "ogg"
+        self.cpp_info.components["ogglib"].names["cmake_find_package_multi"] = "ogg"
+        self.cpp_info.components["ogglib"].libs = tools.collect_libs(self)
```

Comment:
```suggestion
        self.cpp_info.components["ogglib"].libs = ["ogg"]
```

Reply:
What is the advantage? There is only one file, is not always more robust to use tools.collect_libs in this case?

Reply:
**If** ogg would generate multiple libraries, then we will immediately detect this instead of relying on non-guaranteed ordering.
Using `tools.collect_libs` has also its merits.